Resources/string.csv

```diff
@@ -11,735 +20,734 @@
+DictatorInfoLong,(船员阵营):\n当独裁者在会议阶段投票给玩家后，会议会被强制结束并放逐其投票对象。该技能发动的代价是独裁者会自杀。说白了就是一换一。,"(Crewmates):\nWhen you vote someone, the meeting will end on the spot and the player you voted will be ejected.\nThe moment you vote someone out, you will also die.\nWhen you vote for someone in a meeting, they forcibly break that meeting and exile the player they vote for.\nAfter exercising the force, the Dictators die just after meeting.","(Член Экипажа):\nЕсли Диктатор проголосует за любого игрока во время собрания, он сможет принудительно завершить собрание и кикнуть игрока за которого он отдал голос \nТак же Диктатор умрёт после собрания когда он отдаст голос."
+SeerInfoLong,(船员阵营):\n每当玩家死亡时，灵媒会看到杀戮闪烁。,"(Crewmates):\nWhenever someone dies, you will see a ""kill flash"".","(Член Экипажа):\nПровидец видит ''Вспышку Убийства'' каждый раз, когда игроки умирают."
+DetectiveInfoLong,(船员阵营):\n侦探报告尸体后，会收到一条线索消息，该消息会告诉侦探被害者的职业是什么，根据房主设定，侦探甚至可能知道凶手的职业是什么。,"(Crewmate):\nAfter the Detective reports the body, it will receive a hint message that will tell the Detective what the role of the body was. Depending on the settings, the Detective may even know what role the victim's killer has.","(Член Экипажа):\nПосле того, как Детектив зарепортит труп, он получит сообщение с подсказкой, которое сообщит Детективу, какая была роль у трупа. В зависимости от настроек Детектив может даже знать, какая роль у убийцы жертвы"
+UndercoverInfoLong,(船员阵营):\n内鬼会将卧底认作自己的队友，但卧底并不知道内鬼是谁。,"(Crewmate):\nThe Impostors knows who Undercover is and sees him as a teammate, but Undercover himself does not know who the Impostors are.","(Член Экипажа):\nПредатели знают кто является Агентом и видят его как своего товарища по команде, но сам Агент не знает, кто является Предателем."
+NiceGuesserInfoLong,(船员阵营):\n正义赌怪可以在会议时猜测某位玩家的职业，正确则击杀目标，错误则会自杀。\n猜测指令为：/bt [玩家编号] [职业名]\n您可以在玩家名字前看到该玩家的编号，或者使用/id指令查看所有玩家的编号。,"(Crewmate):\nNice Guesser can guess the roles of certain players during an encounter. If he guesses the role correctly, he will kill that target, and if wrong, he will commit suicide. \nGuessing command: ''/bt [Player ID] [Role Name]'' \nYou can see the player ID before the player's nickname, or you can use the ''/id'' command to see the IDs of all players.","(Член Экипажа):\nДобрый Угадыватель может угадывать роли определённого игрока во время встречи. Если он угадает роль правильно, то он убьет эту цель, а если неправильно, то он совершит самоубийство. \nКоманда для угадываний: '/bt [Номер игрока] [Название Роли]'' \nВы можете увидеть номер игрока перед его никнеймом, или же для просмотра номеров всех игроков нужно использовать команду ''/id'' ."
+ChivalrousExpertInfoLong,(船员阵营):\n侠客可以正义击杀任何阵营的人，不过机会只有一次。,"(Crewmate):\nKnight может убить любого игрока с любой ролью, но убить он сможет только один раз за игру.","(Член Экипажа):\nРыцарь может убить любого игрока с любой ролью, но убить он сможет только один раз за игру."

```

Comment:
Did you missed the English here?

Reply:
Yes, I noticed some points where I missed the translation